src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ReadAndUpdateOperationRetryTest.java

```diff
@@ -285,141 +271,79 @@ public void findAndUpdateWithRetryFailure() throws Exception {
-                Values.of(
-                    CustomValueSerializers.getDoubleMapValues(shredDocument.queryNumberValues())),
-                Values.of(
-                    CustomValueSerializers.getStringMapValues(shredDocument.queryTextValues())),
-                Values.of(CustomValueSerializers.getSetValue(shredDocument.queryNullValues())),
-                Values.of(
-                    CustomValueSerializers.getTimestampMapValues(
-                        shredDocument.queryTimestampValues())),
-                Values.of(shredDocument.docJson()),
-                Values.of(CustomValueSerializers.getDocumentIdValue(shredDocument.id())),
-                Values.of(tx_id1))
-            .withColumnSpec(
-                List.of(
-                    QueryOuterClass.ColumnSpec.newBuilder()
-                        .setName("applied")
-                        .setType(TypeSpecs.BOOLEAN)
-                        .build()))
-            .withSerialConsistency(queriesConfig.serialConsistency())
-            // `false` in response for LWT indicate failure
-            .returning(List.of(List.of(Values.of(false))));
-
-    ValidatingStargateBridge.QueryAssert updateRetryFailedQueryAssert =
-        withQuery(
-                collectionUpdateCql,
-                Values.of(CustomValueSerializers.getSetValue(shredDocument.existKeys())),
-                Values.of(CustomValueSerializers.getIntegerMapValues(shredDocument.arraySize())),
-                Values.of(CustomValueSerializers.getStringSetValue(shredDocument.arrayContains())),
-                Values.of(
-                    CustomValueSerializers.getBooleanMapValues(shredDocument.queryBoolValues())),
-                Values.of(
-                    CustomValueSerializers.getDoubleMapValues(shredDocument.queryNumberValues())),
-                Values.of(
-                    CustomValueSerializers.getStringMapValues(shredDocument.queryTextValues())),
-                Values.of(CustomValueSerializers.getSetValue(shredDocument.queryNullValues())),
-                Values.of(
-                    CustomValueSerializers.getTimestampMapValues(
-                        shredDocument.queryTimestampValues())),
-                Values.of(shredDocument.docJson()),
-                Values.of(CustomValueSerializers.getDocumentIdValue(shredDocument.id())),
-                Values.of(tx_id2))
-            .withColumnSpec(
-                List.of(
-                    QueryOuterClass.ColumnSpec.newBuilder()
-                        .setName("applied")
-                        .setType(TypeSpecs.BOOLEAN)
-                        .build()))
-            .withSerialConsistency(queriesConfig.serialConsistency())
-            .returning(List.of(List.of(Values.of(false))));
+    SimpleStatement stmt4 = nonVectorUpdateStatement(shredDocument, tx_id2);
+    List<Row> rows4 = Arrays.asList(resultRow(COLUMNS_APPLIED, 0, Boolean.FALSE));
```

Comment:
Maybe copy this comment from above:

```
 // `false` in response for LWT indicate failure
```

Reply:
oh, let me change them into better variable names.